guides/manual-installation.md

```diff
@@ -77,6 +78,9 @@ firewall-cmd --reload
 
 ### Install postgresql
 
+> Note: This guide was tested with the PostgreSQL version **15.5**
```

Comment:
This 15.5 version looks suspicious. We are already in `16`.
@rtorrero could you confirm this?

Reply:
I just checked in the ansible-installed machine with sles 15 sp5 and I get:
```
localhost:~ # psql --version
psql (PostgreSQL) 15.5
```

I see `postgresql16-server` indeed available to be installed in zypper, but when using the generic package name `postgresql-server` it still pulls `postgresql15-server` by default.

Reply:
I took a quick look at the spec file and per default on sle15 sp5 we get postgresql15-server installed :)

Reply:
@EMaksy But the installed package version on zypper is still `16`. Maybe we need to put a comment about that, or ask to the package maintainer why is this happening.

Reply:
I will ping the maintainer :+1:

Reply:
Just a quick update about the Postgres version: 

The postgresql package is a wrapper package which pulls in the main package and postgresqlXX usually.

The version of the package is unrelated to the version of Postgres which is used.

For SLES15 SP5 it is Postgres version 15.
For SLE15 SP6 it will be Postgres version 16.

So I think we are good to go with 15.5 :+1: